### ISSUE TITLE
Error when updating via WP-CLI

### ISSUE THREAD

**Issue by rilwis** [Automattic/jetpack#14236]
Hi,

I use WP-CLI to update the plugins on our VPS. And when I run `wp plugin update --all --allow-root`, I got this message:

```
PHP Fatal error:  Uncaught Error: Class 'Automattic\Jetpack\Roles' not found in /var/www/fitwp.com/public/plugins/jetpack/vendor/automattic/jetpack-sync/src/Listener.php:270
Stack trace:
#0 /var/www/fitwp.com/public/plugins/jetpack/vendor/automattic/jetpack-sync/src/Listener.php(239): Automattic\Jetpack\Sync\Listener->get_actor('jetpack_plugins...', Array)
#1 /var/www/fitwp.com/public/plugins/jetpack/vendor/automattic/jetpack-sync/src/Listener.php(114): Automattic\Jetpack\Sync\Listener->enqueue_action('jetpack_plugins...', Array, Object(Automattic\Jetpack\Sync\Queue))
#2 /var/www/fitwp.com/public/wp-includes/class-wp-hook.php(288): Automattic\Jetpack\Sync\Listener->action_handler(Array, Array)
#3 /var/www/fitwp.com/public/wp-includes/class-wp-hook.php(312): WP_Hook->apply_filters('', Array)
#4 /var/www/fitwp.com/public/wp-includes/plugin.php(478): WP_Hook->do_action(Array)
#5 /var/www/fitwp.com/public/plugins/jetpack/vendor/automattic/jetpack-sync/src/modules/Plugins.php(168): do_action('jetpack_plugins...', Array, Array)
#6 / in /var/www/fitwp.com/public/plugins/jetpack/vendor/automattic/jetpack-sync/src/Listener.php on line 270
Fatal error: Uncaught Error: Class 'Automattic\Jetpack\Roles' not found in /var/www/fitwp.com/public/plugins/jetpack/vendor/automattic/jetpack-sync/src/Listener.php:270
Stack trace:
#0 /var/www/fitwp.com/public/plugins/jetpack/vendor/automattic/jetpack-sync/src/Listener.php(239): Automattic\Jetpack\Sync\Listener->get_actor('jetpack_plugins...', Array)
#1 /var/www/fitwp.com/public/plugins/jetpack/vendor/automattic/jetpack-sync/src/Listener.php(114): Automattic\Jetpack\Sync\Listener->enqueue_action('jetpack_plugins...', Array, Object(Automattic\Jetpack\Sync\Queue))
#2 /var/www/fitwp.com/public/wp-includes/class-wp-hook.php(288): Automattic\Jetpack\Sync\Listener->action_handler(Array, Array)
#3 /var/www/fitwp.com/public/wp-includes/class-wp-hook.php(312): WP_Hook->apply_filters('', Array)
#4 /var/www/fitwp.com/public/wp-includes/plugin.php(478): WP_Hook->do_action(Array)
#5 /var/www/fitwp.com/public/plugins/jetpack/vendor/automattic/jetpack-sync/src/modules/Plugins.php(168): do_action('jetpack_plugins...', Array, Array)
#6 / in /var/www/fitwp.com/public/plugins/jetpack/vendor/automattic/jetpack-sync/src/Listener.php on line 270
```

Screenshot:

https://prnt.sc/qao8f9

The previous version of Jetpack installed is 7.8. And I was trying to update to 8.0. There are plugins that need to upgrade, too (Meta Box, Slim SEO, Falcon).

All plugins are updated successfully. It just still shows the error after the upgrade process. I think Jetpack is trying to do sync or something and it doesn't work.

**Comment by htdat** [comment]
Hey anh @rilwis 😃

It's a known issue with Jetpack 8.0 https://wordpress.org/support/topic/fatal-error-when-updating-to-jetpack-8-0-read-this/

tl;dr: it's an one-time issue during update and will not cause any issue then.

**Comment by rilwis** [comment]
Hi @htdat ,

It's good to know that! Thanks for your support!
